bootcamp/tutorials/milvus_usages/funnel_search_with_matryoshka.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
<p>This description is kinda verbose.</p><p> </p><p> </p><ul><li>No need to say this. Just something like "we will load ..."</li></ul><p><em>We'll abbreviate the discussion of loading and embeddings the data</em></p><p> </p><p>searching over subsets of embeddings,</p><ul><li>do you mean searching for partial dimensions? that's a weird way to put it.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>